website/src/utils/getTags.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * 
+ * @param tagsDistribution contains the tag list with number of occurrences
+ * @param sortBy can have two values name | count; default set as name
+ * @returns tags
+ * 
+ * ex : getTagsSorted(tagsDistribution,"count");
+ * for this function call the tags will be returned in sorted by count order
+ */
+
+const getTagsSorted = (tagsDistribution: any, sortBy: "name" | "count" = "name") => {
```

Comment:
Should we rename this file to sortTags.ts  to justify its functionality ?

Reply:
tagsDistribution: any, sortBy: "name" | "count" = "name" 
Does this has utility as we are always  sorting alphabetically.

Reply:
i think we can keep this code, if we want to change the tag sort later